projects/laji/src/app/shared-modules/select/select/select.component.ts

```diff
@@ -215,7 +235,12 @@ export class SelectComponent<T extends IdType|SelectOption = string> implements
       });
     });
 
-    this.open = this.open || !!this.selectedOptions.length;
```

Comment:
Looks like the last commit removed little too much as this was here before

Reply:
Good catch, fixed.